Octokit/Models/Response/EventInfo.cs

```diff
@@ -204,6 +204,23 @@ public enum EventInfoState
         /// url of the reference's source.
         /// </summary>
         [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "Crossreferenced")]
-        Crossreferenced
+        Crossreferenced,
+
+        /// <summary>
+        /// The issue was reveiewed.
+        /// </summary>
+        Reviewed,
+
+        /// <summary>
+        /// A line comment was made.
+        /// </summary>
+        [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "Linecommented")]
+        Linecommented,
```

Comment:
@ryangribble I thought we could do something like [this]9https://github.com/octokit/octokit.net/blob/c9b2c1260bc87b7782d4fd9645d43a8885203923/Octokit/Models/Response/Reaction.cs#L10) here?

```csharp
[Parameter(Value = "linecommented")]
LineCommented,
[Parameter(Value = "commitcommented")]
CommitCommented
```